### PR TITLE
add-domain-from-current-URL-to-context-menu-for-following-58

### LINKED ISSUE #58: Add the current URL's domain to the context menu for easy following
ntoll suggested we add the domain of the URL for the page you're on to the context menu so that it's easy to follow.

### PR DESCRIPTION
**Pull request by terrycojones**
ceronman:**approve with comment**

Add follow/goto the page URL's domain to the context menu. Removed passing of unused arg to addContextMenuItem.

Fixes #58

### CODE REVIEW

**Comment by terrycojones** [comment]
Hmm, this seems not to work in 100% of cases. If I open a new tab and go to oreilly.com in it, the context menu says "Goto/Follow newtab". It works when switching between tabs. Investigating....

**Comment by terrycojones** [comment]
OK, fixed the newtab issue.

File: values.js [hunk 1]
```diff
@@ -37,6 +37,47 @@ var valueUtils = {
             return str.toLowerCase();
         }
     },
+    extractDomainFromURL: function(str){
```
Comment (ceronman) [inline values.js]: It'd be better to rename `str` to `url`

**Comment by ceronman** [comment]
+1. Looks good. Just added one comment.

File: values.js (hunk 1 above)
Comment (terrycojones) [inline values.js]: Done.

**Pull request merged**